src/gcp_scanner/gui/static/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/static/logo.png" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>GCP Scanner</title>
+  <script type="module" crossorigin src="/static/assets/index-307c4f89.js"></script>
+  <link rel="stylesheet" href="/static/assets/index-f10fc779.css">
+</head>
+
+<body>
+  <div id="root"></div>
+  
+</body>
+
+</html>
```

Comment:
Add newline

Reply:
files in `src/gcp_scanner/gui/static` are files outputted from the build of react project in `visualization_tool` so it will contain all files needed for the tool such as HTML and images 
we can remove the `src/gcp_scanner/gui/static` folder and add a new step in the `python-publish` workflow to build the vis tool which will create this static folder

Reply:
Yeah let's do that and rebuild the files on demand

Reply:
Removed the folder. and we can update the workflow later on